visualization/js/intro.js/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
```

Comment:
Do we need the entire Introjs folder or just the js/css files? We should only keep the minimum that's needed